test/CMakePassthroughVOLTests.cmake

```diff
@@ -142,7 +142,7 @@ add_custom_target(HDF5_VOLTEST_LIB_files ALL COMMENT "Copying files needed by HD
               -P "${HDF_RESOURCES_DIR}/volTest.cmake"
       )
       set_tests_properties (VOL-${volname}-${voltest} PROPERTIES
-          ENVIRONMENT "srcdir=${HDF5_TEST_BINARY_DIR}/${volname};HDF5TestExpress=${HDF_TEST_EXPRESS}"
```

Comment:
Rather than CMake contending with the possibly user-set HDF5TestExpress env. var, the tests now check if there is a H5_TEST_EXPRESS_LEVEL_DEFAULT macro defined and use that as the initial level if so. Then the user env. var can override that setting.